guacamole/src/main/webapp/app/auth/service/authenticationService.js

```diff
@@ -68,6 +69,17 @@ angular.module('auth').factory('authenticationService', ['$injector',
      */
     var AUTH_STORAGE_KEY = 'GUAC_AUTH';
 
+    /**
+     * List of Logout handlers that are ensured to be run before this service
+     * forces the page to be reloaded.
+     * 
+     * Warning: If a handler has a page reload or redirect in it this can cause
+     * handlers not to be run.
+     * 
+     * @type Array of Angular promises
```

Comment:
This should be `@type Promise[]` or `@type Promise.<WhateverItResolvesWith>[]` if the type of resolution is known. See: https://github.com/apache/guacamole-client/blob/0091bb1aea14c567c8166f0ed8eadf7c31b6bd6e/guacamole/src/main/webapp/app/rest/services/schemaService.js#L45